docs/jade-getting-started.md

```diff
@@ -373,50 +373,28 @@ Ensure that:
 ### Personal Dev Environment Setup
 We're moving away from setting up personal dev environments for every developer. Instead, we are moving
 towards using BEEs ([BEE url](https://beehive.dsp-devops.broadinstitute.org/environments/new), [TDR on BEEs](https://docs.google.com/document/d/1kyjrOKzUthwKu-m38Da2niNEh-IkbUzxtfT29EWw8ag/edit#heading=h.5gsyp5q4qds5))
-However, there are still some use cases for personal dev environments.
+However, there are still some use cases for personal dev environments. So, you should feel free to
+use one of the existing personal dev environments for your work (but check with the data repo team to
+see if anyone else is using a particular environment).
```

Comment:
Note to reviewers: The terraform code to create personal deployments is in the terraform-jade repository (that is now archived). My opinion is that we can just use existing namespace configurations - it doesn't really matter whose name was originally assigned to them.